locales/fa/validation-inline.php

```diff
@@ -13,15 +13,15 @@
 
 return [
     'accepted'             => 'این مقدار باید پذیرفته شده باشد.',
-    'accepted_if'          => 'This field must be accepted when :other is :value.',
+    'accepted_if'          => 'این مقدار باید پذیرفته شود زمانه که :other برابر با :value.',
     'active_url'           => 'این مقدار یک آدرس معتبر نیست.',
     'after'                => 'این مقدار باید یک تاریخ بعد از :date باشد.',
     'after_or_equal'       => 'این مقدار باید یک تاریخ مساوی یا بعد از :date باشد.',
     'alpha'                => 'این مقدار تنها میتواند شامل حروف باشد.',
     'alpha_dash'           => 'این مقدار تنها میتواند شامل حروف، اعداد، خط تیره و زیرخط باشد.',
     'alpha_num'            => 'این مقداز تنها میتواند شامل حروف و اعداد باشد.',
     'array'                => 'این مقدار باید یک آرایه باشد.',
-    'attached'             => 'This field is already attached.',
+    'attached'             => 'این فیلد قبلاً پیوست شده است.',
```

Comment:
The attached key belongs to Laravel Nova and we moved it to another file: https://github.com/Laravel-Lang/lang/blob/master/locales/fa/validation-nova-inline.php#L15

Reply:
ok :)